### PR TITLE
Update for Anki 2.1.50 and Qt6

### PR DESCRIPTION
**Pull request by RisingOrange**

### CODE REVIEW

**Comment by levinuss** [comment]
The fix does not seem to work for me with Anki 2.1.51 and Qt5. Is there a additional change needed for 2.1.51 or am I doing something wrong?

File: src/replacer.js [hunk 1]
```diff
@@ -47,31 +46,52 @@ var insert_symbols = new function() {
...
+        // for Anki >= 2.1.50
+        setTimeout(() => {
+            require("anki/ui").loaded.then(() => {
+                fields = require("anki/NoteEditor").instances[0].fields
+                for (const field of fields) {
+                    field.element.then((fieldElm) => {
+                        if (!fieldElm.hasAttribute("has-type-symbols")) {
+                            const editingArea = fieldElm.getElementsByClassName("editing-area")[0];
+                            const shadowRoot = editingArea.getElementsByClassName("rich-text-editable")[0].shadowRoot;
+                            const editable = shadowRoot?.querySelector("anki-editable");
```
Comment (williamc85) [inline src/replacer.js:78]: Should the '?' be there?

Comment (RisingOrange) [reply src/replacer.js:78]: It works like that for me. Does it not work for you? I think the "?." syntax works only on some platforms. I'll change it.

Comment (williamc85) [reply src/replacer.js:78]: Thanks! When I ran it, it wouldn't work until I removed the '?'

Comment (RisingOrange) [reply src/replacer.js:78]: Good to know. Thanks!

**Comment by AnKingMed** [comment]
@jefdongus this is tested and working. Can you please update this on AnkiWeb?

**Comment by jefdongus** [comment]
Thank you @RisingOrange and @williamc85! I'm not sure why but I didn't get an email notification about this until @AnKingMed tagged me. Sorry for making y'all wait so long...

**Pull request merged**